lib/core/enhanceError.js

```diff
@@ -17,11 +17,11 @@ module.exports = function enhanceError(error, config, code, request, response) {
   }
 
   error.request = request;
-  error.response = response;
+  error.response=response;
```

Comment:
format: style mismatch:
```suggestion
  error.response =response;
```

`33e0cb93` ␣ should be inserted at column 17.